python/test/Ice/asyncio/AllTests.py

```diff
@@ -25,22 +25,22 @@ async def allTestsAsync(helper, communicator):
     sys.stdout.write("testing exceptions... ")
     sys.stdout.flush()
     try:
```

Comment:
Unrelated fix, we were testing the sync versions. Ice.wrap_future was not used ...